### PR TITLE
normal and fast withdrawals

### PR DESCRIPTION
**Pull request by Mayhul-Jindal**

### CODE REVIEW

File: client/withdrawal_normal.go [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+	if err != nil {
+		return ValidateWithdrawalResponse{}, err
+	}
+
+	// todo check if currency there or not
+	coinStatus, err := c.getCoinStatus(ctx)
+	if err != nil {
+		return ValidateWithdrawalResponse{}, err
+	}
+	log.Println("coinstatus", coinStatus)
```
Comment (prathmesh-bhalekar) [inline client/withdrawal_normal.go]: remove all logs

File: client/withdrawal_normal.go [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/tanx-libs/tanx-connector-go/crypto_cpp"
+)
+
+type StartNormalWithdrawalPaylaod struct {
```
Comment (prathmesh-bhalekar) [inline client/withdrawal_normal.go:20]: fix typo

File: client/withdrawal_normal.go [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+
+	bd, err := strconv.Atoi(blockchainDecimal)
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("%f", ToDecimal(balance, bd)), nil
+}
+
+func (c *Client) CompleteNormalWithdrawal(ctx context.Context, rpcURL string, ethPrivateKey string, userPublicEthAddress string, currency Currency) (string, error) {
```
Comment (prathmesh-bhalekar) [inline client/withdrawal_normal.go:278]: what is the use of this function

**Comment by Mayhul-Jindal** [comment]
Specify the versions you need to test for here `go-version: [ '1.21.0', '1.21.1', '1.21.2' ]`
image

**Review by prathmesh-bhalekar — APPROVED** [review]

**Pull request merged**